.travis.yml

```diff
@@ -11,7 +11,8 @@ env:
     - SCAN_WORKSPACE="SwiftSampleProject/SwiftSampleProject.xcworkspace"                 SCAN_SCHEME="SwiftyVokoder"              SCAN_SDK="iphonesimulator"
     - SCAN_WORKSPACE="Paging Data Source Example/Paging Data Source Example.xcworkspace" SCAN_SCHEME="Paging Data Source Example" SCAN_SDK="iphonesimulator"
 script: bundle exec scan
-after_script:
+after_success:
+  - bundle exec pod repo update # Travis isn't finding the latest VOKUtilities release: must have an old repo cached
```

Comment:
Once the build finishes, I'll compare the run time against the last PR build: https://travis-ci.org/vokal/Vokoder/builds/174855777